wand-watcher.lic

```diff
@@ -0,0 +1,153 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#wand-watcher
+=end
+
+no_kill_all
+no_pause_all
+
+custom_require.call(%w[common common-items])
+
+class WandWatcher
+  include DRC
+  include DRCI
+
+  def initialize
+    settings = get_settings();
+    @wand_list = settings.wands
+    @queued_wands = []
+    @no_use_scripts = settings.wand_watcher_no_use_scripts || []
+    @startup_delay = settings.wand_watcher_startup_delay || 10
+    @passive_delay = settings.wand_watcher_passive_delay || 60
+    @activation_failure_messages = [/^The .* remains inert/]
+    UserVars.wand_watcher_timers = {} unless UserVars.wand_watcher_timers
+
+    #don't allow users to make foolish mistakes, always prevent use during go2/burgle
+    @no_use_scripts << 'go2'
+    @no_use_scripts << 'burgle'
```

Comment:
If `go2` or `burgle` are already in the user's `wand_watcher_no_use_scripts ` yaml config, then the original lines 25-26 -- although very succinct -- would duplicate those entries. This tweak is my novice ruby mind's way of only adding the entries if they don't already exist.
```suggestion
    ['go2', 'burgle'].each { |script| @no_use_scripts.push(script) unless @no_use_scripts.include?(script) }
```

Reply:
I think forcing this into a looping construct for such a simple thing is overkill.  
There is minimal impact to having duplicates of the these two scripts in the array, and adding the looping construct just makes the code more complex without extra benefit in my opinion.

Reply:
Yeah, I wasn't really a fan of how much extra code that turned into, either. If there's little performance impact to checking if a script is running multiple times then the duplicates may be ok then.